react/features/base/styles/functions.js

```diff
@@ -136,6 +142,23 @@ export function getRGBAFormat(color: string, alpha: number): string {
     return color;
 }
 
+/**
+ * Decides if a color is light or dark based on the ITU-R BT.709 and W3C
+ * recommendations.
+ *
+ * NOTE: Please see https://www.w3.org/TR/WCAG20/#relativeluminancedef.
+ *
+ * @param {string} color - The color in rgb, rgba or hex format.
+ * @returns {boolean}
+ */
+export function isDarkColor(color: string): boolean {
```

Comment:
❤️

Reply:
Looks like this is not used yet though?

Reply:
Yep, see above, thanks for pointing it out